lms/djangoapps/teams/views.py

```diff
@@ -81,7 +74,38 @@ def team_post_save_callback(sender, instance, **kwargs):  # pylint: disable=unus
                 )
 
 
-class TeamsDashboardView(View):
+class TeamAPIPagination(pagination.PageNumberPagination):
+    """
+    Pagination format used by the teams API.
+    """
+    page_size_query_param = "page_size"
+
+    def get_paginated_response(self, data):
+        """
+        Annotate the response with pagination information.
+        """
+        return Response({
+            'next': self.get_next_link(),
```

Comment:
I thought we'd discovered that "next" and "previous" weren't needed. I suppose it makes sense to include these if we want to use HATEOAS, but again if we're doing this I'd want to see all of our REST APIs follow suit.